### PR TITLE
The newly added Description will appear in Image Details only after the web page is refreshed

### LINKED ISSUE #1575: The newly added Description will appear in Image Details only after the web page is refreshed
### Precondition (*)
1. Install Magento with Adobe Stock Integration
2. Enhanced Media Gallery is Enabled
3. A saved image Preview from Adobe Stock

### Steps to reproduce (*)
1. Go to **Content - Media Gallery**
2. Select an image saved from Adobe Stock and click "three dots" in the bottom right
3. Select **View Details**
the Description field will not be available
no_descr
4. Close Image Details window
5. Select the same image and click "three dots" in the bottom right
6. Select **Edit**
7. Type anything into **Description** field and click **Save** 
8. Select the same image and click "three dots" in the bottom right
9. Select **View Details** 
10. Verify if the _Description_ appeared

### Expected result (*)
The _Description_ appeared

### Actual result (*)
The _Description_ is not displayed until the web page is refreshed

### PR DESCRIPTION
**Pull request by yolouiese**
### Description (*)
- Added a condition if image id is set then retrieval of image details will be triggered as well and not only when image id is undefined. Trigger of image details retrieval will update the image details with newly updated data when viewing image details.

### Fixed Issues (if relevant)
1. Fixes magento/adobe-stock-integration#1575: The newly added Description will appear in Image Details only after the web page is refreshed

### Manual testing scenarios (*)
1. Go to **Content > Media Gallery**
2. Select an image saved from Adobe Stock and click "**three dots**" in the bottom right
3. Select View Details. The Description field will not be available yet.
4. Close Image Details window
5. Select the same image and click "three dots" in the bottom right then select **Edit**
6. Type anything into the Description field and click Save
7. Select the same image and click "three dots" in the bottom right
8. Select View Details
9. Verify if the Description appeared/updated

### CODE REVIEW

**Comment by yolouiese** [comment]
@magento run all tests

**Review by sivaschenko — CHANGES_REQUESTED** [review]
Thanks for the pull request @yolouiese ! What do you think about updating or resetting the outdated details stored in `images[imageId]` after the new details have been saved? That should allow to skip extra request for image details that have not been changed

**Comment by sivaschenko** [comment]
@yolouiese can you please also sign the Adobe CLA - in the checks area underneath

**Comment by yolouiese** [comment]
> Thanks for the pull request @yolouiese ! What do you think about updating or resetting the outdated details stored in `images[imageId]` after the new details have been saved? That should allow to skip extra request for image details that have not been changed

That will do. I'll try to work on that. Thank you @sivaschenko.

**Comment by yolouiese** [comment]
@magento run all tests

File: MediaGalleryUi/view/adminhtml/web/js/image/image-actions.js [hunk 1]
```diff
@@ -74,15 +74,18 @@ define([
         saveImageDetailsAction: function () {
             var saveDetailsUrl = this.mediaGalleryEditDetails().saveDetailsUrl,
                 modalElement = $(this.modalSelector),
-                dataForm = modalElement.find('#image-edit-details-form');
+                imageId = this.imageModel().getSelected().id;
 
-            if (dataForm.validation('isValid')) {
+            modalElement.find('#image-edit-details-form').validation();
```
Comment (sivaschenko) [inline MediaGalleryUi/view/adminhtml/web/js/image/image-actions.js]: @yolouiese Are the validation changes relevant to this pull request?

Comment (yolouiese) [reply MediaGalleryUi/view/adminhtml/web/js/image/image-actions.js]: Sorry, it was not supposed to be included in the pull request. Maybe it got included after I did git pull origin to update my branch. I'll update again my pull request.

**Comment by yolouiese** [comment]
@magento run all tests

**Comment by yolouiese** [comment]
@magento run all tests

**Comment by sivaschenko** [comment]
@magento run Functional Tests CE

**Review by sivaschenko — APPROVED** [review]

**Comment by chalov-anton** [comment]
:heavy_check_mark: QA Passed 

The newly added _Name_ and _Description_ is displayed without refreshing the web page
Peek 2020-07-14 16-03

**Pull request merged**